### DISCUSSION TITLE
Playtime: Be able to rearrange tracks by dragging anywhere in mixer control panel at the bottom (not only top)

### DISCUSSION THREAD

**Discussion by joungle123** [helgoboss/helgobox#1248] [Ideas]
It would be fantastic to rearrange tracks by simply dragging mixer tracks (bottom of Playtime 2) and not only just the top of the column, it would be highly convenient

**Comment by helgoboss** [comment]
This is already possible. You can drag and drop the header of the track panel, not just the column panel. Or are you talking about something else?

**Reply by joungle123** [reply]
@helgoboss well yes, I see that it is possible to drag when clicking on the header at the bottom. My suggestion was to make it possible to click on the mixer below the column and move it anywhere instead of trying to click just the header. In other words - drag and drop by clicking and dragging the whole column anywhere and not just the header (of course excluding icons, buttons and slider - just any area of the column and not just headers). Do you get the idea?

**Reply by helgoboss** [reply]
Okay, this is possible. I will test this at some point and check whether it feels good.

**Reply by joungle123** [reply]
Thank you Helgo!
